libopendroneid/opendroneid.h

```diff
@@ -145,14 +153,15 @@ typedef struct {
 
 typedef struct {
     ODID_status_t Status;
-    float SpeedNS;            // m/s
-    float SpeedEW;            // m/s
+    float Direction;          // Degrees
+    float SpeedHorizontal;    // m/s. Positive only
     float SpeedVertical;      // m/s
     double Latitude;
     double Longitude;
     float AltitudeBaro;       // meter
```

Comment:
MSL?

Reply:
The specification says "(Ref 29.92 inHg, 1013.24 mb)". I will add that.